app/schemas/userDataSanitizationSchema.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
+import {
+  parseAndCleanInput,
+  removeExtraWhiteSpaces,
+  sanitizeAndTrimText,
+} from '../utils/formInputSanitization';
+
+const userDataSanitizationSchema = z.object({
+  name: z.string().transform(removeExtraWhiteSpaces),
+  email: z.string().trim().toLowerCase(),
```

Comment:
We're not performing sanitization on the email address here - is this because that's essentially already being handled in `userDataValidationSchema`?

Reply:
True and we don't want to remove unnecessary symbols or characters from an email esp if we plan to verify users' emails.